apps/ledger-live-desktop/src/renderer/modals/SignTransaction/Body.tsx

```diff
@@ -49,16 +49,13 @@ type Props = {
   params: Params;
   setError: (error?: Error) => void;
 };
-function useSteps(canEditFees = false, typeTransaction = ""): St[] {
+function useSteps(canEditFees = false): St[] {
   const { t } = useTranslation();
   return useMemo(() => {
     const steps: St[] = [
       {
         id: "summary",
-        label:
-          typeTransaction === "Approve"
-            ? t("send.steps.reviewSummary.title")
-            : t("send.steps.summary.title"),
```

Comment:
Is this removed because we only needed it for the wallet-api integration ?
We might want to make sure we don't break the platform-sdk part tho, I don't know what component it uses

Reply:
It's not removed per se but instead put back to a non-ternary form: 

https://github.com/LedgerHQ/ledger-live/pull/6205/files/5359b50f963b3efd669960afbcaffb8692187ddc#diff-39ce21d822cea075fe02e6bd016907e5a1fc4c0da01380cc36efd52c3115bd1eL72-L78 

In all cases of a transaction signature the text is now the same for the device step (_Connect Ledger_)